tests/jenkins/run_neuron.sh

```diff
@@ -13,12 +13,12 @@ cd $WORKSPACE/${TEST_DIR}
 if [ "${TEST_DIR}" = "testcorenrn" ]; then
     mkdir test${TEST}dat
     mkdir ${TEST}
-    mpirun -n ${MPI_RANKS} ./x86_64/special --mpi -c sim_time=100 test${TEST}.hoc
+    mpirun -n ${MPI_RANKS} ./x86_64/special -mpi -c sim_time=100 test${TEST}.hoc
```

Comment:
I know this was already there, but `mpiexec` is guaranteed to exist in mpi implementations, not mpirun. Besides there were these CPU affinity problem with mpirun.

Reply:
#250